assets/js/components/includes/DropdownFilter.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <select @change="getIndex(items)">
-  <option>
-  {{title}}
-  </option>
-    <option :selected="item.id == getFilterValue()"
-            :id="item.id"
```

Comment:
you shouldn't be interested in the `id` of the html element. not on an option. use `value`, and use the standard way to access a select's value.
besides, think of the great potential for dom-level id clash when you set the id to this, and the great mayhem ensuing when someone does getElementById('EEA')...